<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralWidget">
            <widget class="QPushButton" name="pbVecNrSound">
                <property name="geometry">
                    <rect>
                        <x>100</x>
                        <y>170</y>
                        <width>201</width>
                        <height>51</height>
                    </rect>
                </property>
                <property name="text">
                    <string>vector  noise reduced</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pbVecSound">
                <property name="geometry">
                    <rect>
                        <x>100</x>
                        <y>100</y>
                        <width>201</width>
                        <height>51</height>
                    </rect>
                </property>
                <property name="text">
                    <string>vector sound</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pbOriginSound">
                <property name="geometry">
                    <rect>
                        <x>100</x>
                        <y>30</y>
                        <width>201</width>
                        <height>51</height>
                    </rect>
                </property>
                <property name="cursor">
                    <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="text">
                    <string>Origin sound</string>
                </property>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menuBar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>400</width>
                    <height>22</height>
                </rect>
            </property>
        </widget>
        <widget class="QToolBar" name="mainToolBar">
            <attribute name="toolBarArea">
                <enum>TopToolBarArea</enum>
            </attribute>
            <attribute name="toolBarBreak">
                <bool>false</bool>
            </attribute>
        </widget>
        <widget class="QStatusBar" name="statusBar"/>
    </widget>
    <layoutdefault spacing="6" margin="11"/>
    <resources/>
    <connections/>
</ui>
